lib/round-tracker.js

```diff
@@ -0,0 +1,78 @@
+import { createMeridianContract } from './meridian/contract.js'
+
+export async function createRoundGetter (pgClient) {
+  const contract = await createMeridianContract()
+
+  let sparkRound
+  const updateSparkRound = async (meridianRound) => {
+    sparkRound = await mapCurrentMeridianRoundToSparkRound({
+      meridianContractAddress: contract.address,
+      meridianRound: BigInt(meridianRound),
+      pgClient
+    })
+    console.log('SPARK round started: %s', sparkRound)
+  }
+
+  contract.on('RoundStart', updateSparkRound)
+  updateSparkRound(await contract.currentRoundIndex())
+
+  return () => sparkRound
+}
+
+export async function mapCurrentMeridianRoundToSparkRound ({
+  meridianContractAddress,
+  meridianRound,
+  pgClient
+}) {
+  let sparkRound
+
+  const { rows: [lastMeridianRecord] } = await pgClient.query(
```

Comment:
```suggestion
  const { rows: [lastSparkRoundContract] } = await pgClient.query(
```

I think this fits the new names better

Reply:
I see. Let me improve this a bit more to `contractVersionOfPreviousSparkRound`.